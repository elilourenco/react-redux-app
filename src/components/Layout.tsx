import { Box, Container } from "@mui/material";
import React from "react";



export default function Layout({children}:{children:React.ReactNode}){
    return(
        <>
        <Box>
            <Container maxWidth="lg"> {children}

            </Container>
        </Box>
        </>
    )

}