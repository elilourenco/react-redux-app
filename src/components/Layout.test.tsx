import { render } from "@testing-library/react";
import Layout from "./Layout";


describe("Layout",()=>{
    it("Should  render correctly",()=>{

        const {asFragment} = render(
        <Layout>
            <div>test</div>
        </Layout>);

         expect( asFragment()).toMatchSnapshot()
    })

})