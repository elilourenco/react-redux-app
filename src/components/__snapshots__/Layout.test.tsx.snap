// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Should  render correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1g3nlq1-MuiContainer-root"
    >
       
      <div>
        test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
